Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/common/taimaba/TaimabaEndpoints.java

```diff
@@ -88,4 +89,16 @@ public HttpUrl pages(Board board) {
     public HttpUrl reply(Loadable loadable) {
         return sys.builder().s(loadable.board.code).s("taimaba.pl").url();
     }
+
+    @Override
+    public HttpUrl report(Post post) {
+        return report.newBuilder()
+            .addPathSegment("narcbot")
+            .addPathSegment("ajaxReport.jsp")
+            .addQueryParameter("postId", String.valueOf(post.no))
+            .addQueryParameter("reason", "RULE_VIOLATION")
+            .addQueryParameter("note", "Kuroba generated report")
```

Comment:
You might want to avoid using something so easily traceable to this application, in case the admins don't like it and end up banning the user agent that sent the report. Because reports don't have a captcha on them, that can end up being really abused.

Reply:
should i revert the whole commit, or just edit the note part?

Reply:
Just edit the note part so it's more a generic report. Seems like the note is meant to be an explanation of why it's breaking the rules anyways.